src/config/syscheck-config.c

```diff
@@ -1595,6 +1610,10 @@ int Read_Syscheck(const OS_XML *xml, XML_NODE node, void *configp, __attribute__
                             os_free(tag);
                             return OS_INVALID;
                         }
+                    } else if (strcmp(node[i]->attributes[j], xml_restrict) == 0) {
+                        os_free(restrictfile);
+                        os_strdup(node[i]->values[j], restrictfile);
+                        str_lowercase(restrictfile);
```

Comment:
We are not treating registry entries as case insensitive, please remove this `str_lowercase`.